client/commonFramework/show-completion.js

```diff
@@ -47,13 +47,23 @@ window.common = (function(global) {
           next();
         });
 
+      let timezone = 'UTC';
+      try {
+        timezone = moment.tz.guess();
```

Comment:
Will this overwrite timezone if there is an error, or will the error happen before the assignment?